kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/BluetoothAdvertisementData.java

```diff
@@ -0,0 +1,75 @@
+package org.eclipse.kura.bluetooth;
```

Comment:
1. Move to ``org.eclipse.kura.bluetooth.listener`` since it's only used there
2. Missing License header